src/sqlfluff/dialects/dialect_sqlite.py

```diff
@@ -29,9 +41,240 @@
+                ),
+                Ref("Expression_C_Grammar"),
+            ),
+        ),
+        AnyNumberOf(
+            OneOf(
+                Sequence(
+                    OneOf(
+                        Sequence(
+                            Ref.keyword("NOT", optional=True),
+                            Ref("LikeGrammar"),
+                        ),
+                        Sequence(
+                            Ref("BinaryOperatorGrammar"),
+                            Ref.keyword("NOT", optional=True),
+                        ),
+                        # We need to add a lot more here...
+                    ),
+                    Ref("Expression_C_Grammar"),
+                    Sequence(
+                        Ref.keyword("ESCAPE"),
+                        Ref("Expression_C_Grammar"),
+                        optional=True,
+                    ),
+                ),
+                Sequence(
+                    Ref.keyword("NOT", optional=True),
+                    "IN",
+                    Bracketed(
+                        OneOf(
+                            Delimited(
+                                Ref("Expression_A_Grammar"),
+                            ),
+                            Ref("SelectableGrammar"),
+                            ephemeral_name="InExpression",
+                        )
+                    ),
+                ),
+                Sequence(
+                    Ref.keyword("NOT", optional=True),
+                    "IN",
+                    Ref("FunctionSegment"),  # E.g. UNNEST()
+                ),
+                Sequence(
+                    "IS",
+                    Ref.keyword("NOT", optional=True),
+                    Ref("IsClauseGrammar"),
+                ),
+                Ref("IsNullGrammar"),
+                Ref("NotNullGrammar"),
+                Ref("CollateGrammar"),
+                Sequence(
+                    # e.g. NOT EXISTS, but other expressions could be met as
+                    # well by inverting the condition with the NOT operator
+                    "NOT",
+                    Ref("Expression_C_Grammar"),
+                ),
+                Sequence(
+                    Ref.keyword("NOT", optional=True),
+                    "BETWEEN",
+                    Ref("Expression_B_Grammar"),
+                    "AND",
+                    Ref("Expression_A_Grammar"),
+                ),
+            )
+        ),
+    ),
 )
 
 
+class SetOperatorSegment(BaseSegment):
+    """A set operator such as Union, Minus, Except or Intersect."""
+
+    type = "set_operator"
+    match_grammar: Matchable = OneOf(
+        Sequence("UNION", OneOf("DISTINCT", "ALL", optional=True)),
+        Sequence(
+            OneOf(
+                "INTERSECT",
+                "EXCEPT",
+            ),
+            Ref.keyword("ALL", optional=True),
+        ),
+        exclude=Sequence("EXCEPT", Bracketed(Anything())),
+    )
```

Comment:
This is one is a good candidate for `.copy(remove=...)` too. I think you're just removing `MINUS` right?